### PR TITLE
docs(k8s): add information for k8s v1.27

### PR DESCRIPTION
**Pull request by bene2k1**
### Your checklist for this pull request

- [ ] Check that the commit messages match our requested structure.
- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description
Please describe your pull request.

### CODE REVIEW

File: containers/kubernetes/reference-content/version-support-policy.mdx [hunk 1]
```diff
@@ -37,24 +37,28 @@ Before opening a support ticket related to cluster behavior or a Kubernetes comp
...
 ## Version information
 For a complete changelog update, refer to the official [Kubernetes changelog](https://kubernetes.io/releases/).
 
 The following documentation lists only the main feature changes for each version and provides a link to the version release announcement on Kubernetes' official blog.
 
+### 1.27
+* Official [release announcement](https://kubernetes.io/blog/2023/04/11/kubernetes-v1-27-release/)
+
+**Main changes:**
+* This release consist of 60 enhancements. 18 of those enhancements are entering Alpha, 29 are graduating to beta, and 13 are graduating to stable.
```
Comment (nedaehm) [inline containers/kubernetes/reference-content/version-support-policy.mdx:61]:
```suggestion
* This release consists of 60 enhancements. 18 of those enhancements are entering Alpha, 29 are graduating to beta, and 13 are graduating to stable.
```

File: faq/kubernetes.mdx [hunk 2]
```diff
@@ -63,6 +63,7 @@ Depending on the type of node selected for your pool, one or two types of volume
 
 `kapsule-agent` is a software developed internally at Scaleway. It is used to assist the managing of nodes lifecycles. Its primary function is to set up the network during the initial boot, and toggle the SSH server on or off based on a k8s label of each node.
```
Comment (nedaehm) [inline faq/kubernetes.mdx:64]:
```suggestion
`kapsule-agent` is a software developed internally at Scaleway. It is used to assist in managing nodes lifecycles. Its primary function is to set up the network during the initial boot, and toggle the SSH server on or off based on a k8s label of each node.
```

File: faq/kubernetes.mdx [hunk 3]
```diff
@@ -63,6 +63,7 @@ Depending on the type of node selected for your pool, one or two types of volume
...
 
-## Why cannot I connect to my nodes using SSH?
+## How do I connect to my nodes using SSH?
 
-Kubernetes Kapsule manages your nodes. To ensure the behavior of your cluster, it is not possible to perform actions directly on the nodes. There is no need to connect to them directly, as all actions and configurations should be done using Kubernetes (from the Scaleway Console, `kubectl` command, or Scaleway Kapsule `APIs`).
\ No newline at end of file
+By default, an SSH server is installed and configured on the nodes of your Kubernetes Kapsule cluster. To ensure the correct behavior of your cluster, it is not recommended to perform actions directly on the nodes. All actions and configurations should be done using Kubernetes (from the Scaleway Console, `kubectl` command, or Scaleway Kapsule `APIs`).
+To improve the security of your Kapsule cluster, you can disable this SSH server on your nodes by following the documentation [how to enable or disable SSH on Kubernetes Kapsule nodes](/containers/kubernetes/how-to/enable-disable-ssh/).
```
Comment (nedaehm) [inline faq/kubernetes.mdx:69]:
```suggestion
To improve the security of your Kapsule cluster, you can disable this SSH server on your nodes by following the documentation on [how to enable or disable SSH on Kubernetes Kapsule nodes](/containers/kubernetes/how-to/enable-disable-ssh/).
```

File: containers/kubernetes/reference-content/version-support-policy.mdx [hunk 4]
```diff
@@ -37,24 +37,29 @@ Before opening a support ticket related to cluster behavior or a Kubernetes comp
...
+| 1.25               | August 2022      | October 2023         | February 6, 2023             | February 2025    | March 15, 2025    |
+| 1.24               | April 2022       | July 2023            | April 2022                   | April 2024       | May 15, 2024      |
+| 1.23               | December 8, 2021 | February 2023        | December 9, 2021             | December 2023    | February 15, 2024 |
+| 1.22               | August 4, 2021   | October 2022         | August 11, 2021              | August 2023      | October 15, 2023  |
+| 1.21               | April 8, 2021    | June 2022            | April 30, 2021               | April 2023       | June 15, 2023     |
 
 ## Version information
 For a complete changelog update, refer to the official [Kubernetes changelog](https://kubernetes.io/releases/).
 
 The following documentation lists only the main feature changes for each version and provides a link to the version release announcement on Kubernetes' official blog.
```
Comment (farhakennar) [inline containers/kubernetes/reference-content/version-support-policy.mdx:55]:
```suggestion
The following documentation only lists the main feature changes for each version and provides a link to the version release announcement on Kubernetes' official blog.
```

File: containers/kubernetes/reference-content/version-support-policy.mdx [hunk 5]
```diff
@@ -37,24 +37,29 @@ Before opening a support ticket related to cluster behavior or a Kubernetes comp
...
 
+### 1.27
+* Official [release announcement](https://kubernetes.io/blog/2023/04/11/kubernetes-v1-27-release/)
+
+**Main changes:**
+* This release consists of 60 enhancements. 18 of those enhancements are entering Alpha, 29 are graduating to beta, and 13 are graduating to stable.
```
Comment (farhakennar) [inline containers/kubernetes/reference-content/version-support-policy.mdx:61]: IMO, Alpha/Beta is either in upper or lowercase

**Review by farhakennar — APPROVED** [review]

File: containers/kubernetes/reference-content/version-support-policy.mdx (hunk 5 above)
Comment (bene2k1) [reply containers/kubernetes/reference-content/version-support-policy.mdx:61]:
```suggestion
* This release consists of 60 enhancements. 18 of those enhancements are entering alpha, 29 are graduating to beta, and 13 are graduating to stable.
```

**Pull request merged**